lib/vtls/schannel_verify.c

```diff
@@ -481,6 +518,12 @@ CURLcode Curl_verify_host(struct Curl_cfilter *cf,
   size_t hostlen = strlen(conn_hostname);
   DWORD len = 0;
   DWORD actual_len = 0;
+  PCERT_ALT_NAME_INFO alt_name_info = NULL;
+  DWORD alt_name_info_size = 0;
+  BYTE ip_blob[32] = { 0 };
```

Comment:
how about a plain `unsigned char` here instead of `BYTE` to try to stick to how we write code elsewhere?

Reply:
There's no consistency in the original curl code, it's all over the place. But we can do that. Same applies to coding style and naming conventions.